src/util/psbt/map/output.rs

```diff
@@ -102,8 +102,10 @@ impl TapTree {
         self.0.branch()[0].as_ref().expect("from_inner only parses is_complete builders")
     }
 
-    /// Convert a [`TaprootBuilder`] into a tree if it is complete binary tree.
-    /// Returns the inner as Err if it is not a complete tree
+    /// Converts a [`TaprootBuilder`] into a tree if it is complete binary tree.
+    ///
+    /// # Return
```

Comment:
nit: AFAIR it is usually named "Returns"

Reply:
doh, you got me! I'm going to leave it as is for 2 reasons
- Constant docs fixes are high on my priority list while I'm finding my feet in this project, so this will get done
- This PR has been doing cycling for ages and now has 2 acks, I'll do a follow up

Reply:
no, for sure do not update it discarding reviews

Reply:
Cheers